### DISCUSSION TITLE
When Each user requests the dashboard,Superset can load their own interested filters which are configured by themselves

### DISCUSSION THREAD

**Discussion by wyc990122** [apache/superset#22811] [Q&A / Help] [answered]
==Background:
- I'm a freshman to use superset.I have configured some dashboards which are predefinded the same dashboard filters for my users in my company.
- Recently they have some demands to configure their own filter conditions for every dashboard,and they hope these filter conditions can be loaded while they request these dashboards.
- I have search the similar keywords in the Issues,and I don't know if the superset supports this function.
==Feature Request:
- Each user can configure their own dashboard filter conditions for each dashboard, and it can be loaded while they request the dashboard instead of the same filter conditions configured by the dashboard developers.

Hoping for your replies.Thanks!

**Comment by rusackas** [comment, marked as answer]
You might be able to use Jinja templating to accomplish this goal. @shreesham wrote [this article](https://preset.io/blog/intro-jinja-templating-apache-superset/) on the topic, and may be able to help further.

Also pinging @kasiazjc because this might be an interesting use case for a future re-invention of something like Filter Sets when the time is right.

**Reply by wyc990122** [reply]
> You might be able to use Jinja templating to accomplish this goal. @shreesham wrote [this article](https://preset.io/blog/intro-jinja-templating-apache-superset/) on the topic, and may be able to help further.
> 
> Also pinging @kasiazjc because this might be an interesting use case for a future re-invention of something like Filter Sets when the time is right.

Thanks for your replies,I'll try to learn using jinja templates to accomplish it. And it's better that Superset can natively support this feature just like you mentioned.
